lib/streamlit/__init__.py

```diff
@@ -248,22 +247,22 @@ def write(*args, **kwargs):
 
         Arguments are handled as follows:
 
-        - write(string)     : Prints the formatted Markdown string, with
-          support for LaTeX expression and emoji shortcodes.
-          See docs for st.markdown for more.
-        - write(data_frame) : Displays the DataFrame as a table.
-        - write(error)      : Prints an exception specially.
-        - write(func)       : Displays information about a function.
-        - write(module)     : Displays information about the module.
-        - write(dict)       : Displays dict in an interactive widget.
-        - write(obj)        : The default is to print str(obj).
-        - write(mpl_fig)    : Displays a Matplotlib figure.
-        - write(altair)     : Displays an Altair chart.
-        - write(keras)      : Displays a Keras model.
-        - write(graphviz)   : Displays a Graphviz graph.
-        - write(plotly_fig) : Displays a Plotly figure.
-        - write(bokeh_fig)  : Displays a Bokeh figure.
-        - write(sympy_expr) : Prints SymPy expression using LaTeX.
+            - write(string)     : Prints the formatted Markdown string, with
+              support for LaTeX expression and emoji shortcodes.
+              See docs for st.markdown for more.
+            - write(data_frame) : Displays the DataFrame as a table.
+            - write(error)      : Prints an exception specially.
+            - write(func)       : Displays information about a function.
+            - write(module)     : Displays information about the module.
+            - write(dict)       : Displays dict in an interactive widget.
+            - write(obj)        : The default is to print str(obj).
+            - write(mpl_fig)    : Displays a Matplotlib figure.
+            - write(altair)     : Displays an Altair chart.
+            - write(keras)      : Displays a Keras model.
+            - write(graphviz)   : Displays a Graphviz graph.
+            - write(plotly_fig) : Displays a Plotly figure.
+            - write(bokeh_fig)  : Displays a Bokeh figure.
+            - write(sympy_expr) : Prints SymPy expression using LaTeX.
```

Comment:
Same here: too much indentation.

Reply:
Same reasoning, the aesthetics of having it moved right a bit further to delineate the arguments better

Reply:
Reverted it back to less indentation